src/Sylius/Component/Core/Model/Address.php

```diff
@@ -28,6 +29,7 @@ public function getCustomer(): ?BaseCustomerInterface
 
     public function setCustomer(?BaseCustomerInterface $customer): void
     {
+        Assert::nullOrIsInstanceOf($customer, CustomerInterface::class);
```

Comment:
Shouldn't it be changed in phpdoc instead, above the `Address::customer` property?